meshers/blocky/voxel_library.cpp

```diff
@@ -269,7 +269,7 @@ void VoxelLibrary::generate_side_culling_matrix() {
 
 			CRASH_COND(pattern == nullptr);
 
-			if (full_side_pattern_index == -1 && bitmap.all()) {
+			if (full_side_pattern_index == 0xFFFFFFFF && bitmap.all()) {
```

Comment:
The warning is here. It doesn't compare about assigning -1, only comparing uint vs -1.